SpatialGDK/Source/SpatialGDK/Public/SpatialGDKSettings.h

```diff
@@ -207,4 +207,22 @@ class SPATIALGDK_API USpatialGDKSettings : public UObject
 
 	UPROPERTY(EditAnywhere, config, Category = "Load Balancing", meta = (EditCondition = "bEnableUnrealLoadBalancer"))
 	TSubclassOf<class UAbstractLBStrategy> LoadBalanceStrategy;
+
+	UPROPERTY(EditAnywhere, Config, Category = "Replication", meta = (DisplayName = "Use RPC Ring Buffers"))
+	bool bUseRPCRingBuffers;
+
+private:
+	UPROPERTY(EditAnywhere, Config, Category = "Replication", meta = (EditCondition = "bUseRPCRingBuffers", DisplayName = "Default RPC Ring Buffer Size"))
```

Comment:
nit: these display names are redundant

Reply:
Not quite. This is what it looks like without specifying the display name:
![image](https://user-images.githubusercontent.com/32096431/69535747-7e6e4400-0f74-11ea-8f34-961470da29de.png)

Reply:
The horror